db/migrate/20170118232430_create_join_table_authentication_configuration_script_base.rb

```diff
@@ -0,0 +1,9 @@
+class CreateJoinTableAuthenticationConfigurationScriptBase < ActiveRecord::Migration[5.0]
+  def change
+    create_table :authentication_configuration_script_bases do |t|
```

Comment:
Based on the talk article, the class name will be SingularSingular `AuthenticationConfigurationScriptBase` and its tablename is therefore `authentication_configuration_script_bases`.